### PR TITLE
Fix hyperlinks in messages.xml

### PR DESCRIPTION
**Pull request by orihalcon128**
Fix hyperlinks in messages.xml.

### CODE REVIEW

File: spotbugs/etc/messages.xml [hunk 1]
```diff
@@ -8288,10 +8288,11 @@ String constructComponentName() {
           (and hopefully better) static analysis technique.
           We are interested is getting feedback about the
           usefulness of this bug pattern.
-          To send feedback:
+          To send feedback, either:
           </p>
           <ul>
-            <li>file an issue: <a href="https://github.com/spotbugs/spotbugs/issues">https://github.com/spotbugs/spotbugs/issues</a></li>
+            <li><a href="https://github.com/spotbugs/spotbugs/blob/master/CONTRIBUTING.md">reporting a problem</a></li>
```
Comment (iloveeclipse) [inline spotbugs/etc/messages.xml]: "reporting a problem" -> "contributing guideline"

File: spotbugs/etc/messages.xml [hunk 2]
```diff
@@ -8288,10 +8288,11 @@ String constructComponentName() {
           (and hopefully better) static analysis technique.
           We are interested is getting feedback about the
           usefulness of this bug pattern.
-          To send feedback:
+          To send feedback, either:
```
Comment (iloveeclipse) [inline spotbugs/etc/messages.xml]: "To send feedback, either:" -> "For sending feedback, check:"

File: spotbugs/etc/messages.xml (hunk 1 above)
Comment (iloveeclipse) [reply spotbugs/etc/messages.xml]: Same also below.

File: spotbugs/etc/messages.xml (hunk 2 above)
Comment (orihalcon128) [reply spotbugs/etc/messages.xml]: OK.

File: spotbugs/etc/messages.xml (hunk 1 above)
Comment (orihalcon128) [reply spotbugs/etc/messages.xml]: OK.

**Pull request merged**